wiki/Staff_Log/2020/en.md

```diff
@@ -318,6 +318,7 @@ Abbreviations for user groups are used throughout this log:
 
 - 2020-08-05: Added [Ayucchi](https://osu.ppy.sh/users/7823498) to **Probationary BN**
 - 2020-08-05: Added [Hobbes2](https://osu.ppy.sh/users/8157492) to **Probationary BN**
+- 2020-08-07: Added [Scotty](https://osu.ppy.sh/users/11085809) to **NAT**
```

Comment:
```suggestion
```

Reply:
Don't know how to suggest changes above this point, but NAT additions should be outside the Beatmap Nominators section

Reply:
thanks for pointing it out, fixed it